client/lib/lib.rs

```diff
@@ -80,29 +79,28 @@ pub fn put_deploy(
 /// [`sign_deploy_file()`](fn.sign_deploy_file.html) and then sent to the network for execution
 /// using [`send_deploy_file()`](fn.send_deploy_file.html).
 ///
-/// * `maybe_output_path` specifies the output file, or if empty, will print it to `stdout`. If the
-///   file already exists, it will be overwritten.
+/// * `maybe_output_path` specifies the output file, or if empty, will print it to `stdout`.
 /// * `deploy_params` contains deploy-related options for this `Deploy`. See
 ///   [`DeployStrParams`](struct.DeployStrParams.html) for more details.
 /// * `session_params` contains session-related options for this `Deploy`. See
 ///   [`SessionStrParams`](struct.SessionStrParams.html) for more details.
 /// * `payment_params` contains payment-related options for this `Deploy`. See
 ///   [`PaymentStrParams`](struct.PaymentStrParams.html) for more details.
+/// * If `force` is true, existing files will be overwritten. If `force` is false and any of the
+/// files exist, [`Error::FileAlreadyExists`](../enum.Error.html#variant.FileAlreadyExists) is
+/// returned and no files are written.
```

Comment:
This was copied verbatim from a function where multiple output files are used.  In this case it's a single output file.  Also, the generated link is dead, since it points to `../` which is incorrect here.

```suggestion
/// * If `force` is true, and the output file exists, it will be overwritten. If `force` is false
///   and the output file exists,
///   [`Error::FileAlreadyExists`](enum.Error.html#variant.FileAlreadyExists) is returned and the
///   output file is left unchanged.
```

This change should be applied to `sign_deploy_file()` and `make_transfer()` too.